x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/jwt/JwkSetLoader.java

```diff
@@ -138,18 +137,17 @@ void loadInternal(final ActionListener<Tuple<Boolean, JwksAlgs>> listener) {
         }
     }
 
-    private Tuple<Boolean, JwksAlgs> handleReloadedContentAndJwksAlgs(byte[] bytes) {
+    private JwksAlgs handleReloadedContentAndJwksAlgs(byte[] bytes) {
```

Comment:
we no longer read the isUpdated from the value returned here (it only the the threads view of if it updated the file which is not relevant in a concurrent environment and workflow this used)